browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -1052,26 +1052,41 @@ test.describe('program creation', () => {
 
   test('create common intake form', async ({page, adminPrograms}) => {
     await loginAsAdmin(page)
-
     const programName = 'Apc program'
-    await adminPrograms.addProgram(programName)
-    await adminPrograms.goToProgramDescriptionPage(programName)
 
-    await validateScreenshot(
-      page,
-      'program-description-page-with-intake-form-false',
-    )
+    await test.step('create new program that is not an intake form', async () => {
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.goToProgramDescriptionPage(programName)
+
+      await validateScreenshot(
+        page,
+        'program-description-page-with-intake-form-false',
+      )
+    })
+
     const commonIntakeFormInput = adminPrograms.getCommonIntakeFormToggle()
-    await expect(commonIntakeFormInput).not.toBeChecked()
 
-    await adminPrograms.clickCommonIntakeFormToggle()
-    await validateScreenshot(
-      page,
-      'program-description-page-with-intake-form-true',
-    )
-    await expect(commonIntakeFormInput).toBeChecked()
-    await adminPrograms.submitProgramDetailsEdits()
-    await adminPrograms.expectProgramBlockEditPage(programName)
+    await test.step('expect common intake toggle not to be checked', async () => {
+      await expect(commonIntakeFormInput).not.toBeChecked()
+    })
+
+    await test.step('click common intake toggle and expect it to be checked', async () => {
+      await adminPrograms.clickCommonIntakeFormToggle()
+      await validateScreenshot(
+        page,
+        'program-description-page-with-intake-form-true',
+      )
+      await expect(commonIntakeFormInput).toBeChecked()
+    })
+
+    await test.step('expect application steps to be disabled', async () => {
+      await adminPrograms.expectApplicationStepsDisabled()
```

Comment:
This is the only change functional change I made to this test. The rest of the changes are me splitting it up into `test.step`